bundles/apicurio-registry-operator.v1.0.0-v2.0.0.final/manifests/apicurio-registry-operator_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
Just a question to help my understanding: why was it necessary to add all of these manifest and bundle files? why were they not there in the first place?

Reply:
I added the scenario for we are able to add a new test to ensure that after the change we not only break anything as it is working as supposed to be. 

>

Reply:
Most of these files don't directly impact the test at all.

@camilamacedo86 could you minimize the set of files added in this PR. This should only require CSVs and annotation files, right?